src/utils/path.ts

```diff
@@ -165,7 +165,8 @@ export const normalizeFsPath = (p: string) => normalizePath(p.split('?')[0].repl
 export const normalizeFsPathQuery = (importPath: string) => {
   const pathParts = importPath.split('?');
   const filePath = normalizePath(pathParts[0]);
-  const ext = filePath.split('.').pop().toLowerCase();
+  const filePathParts = filePath.split('.');
+  const ext = filePathParts.length > 1 ? filePathParts.pop()!.toLowerCase() : null;
```

Comment:
are we pretty certain nothing depends on that old behavior?

Reply:
The only thing I could think of is if dot files (e.g. `.eslintrc`) - but I'm not entirely sure/will leave that to @tanner-reits 😉

Reply:
I don't think this change will mess with anything. This function is only used in a single Rollup plugin where we only use the extension for two cases:
1. If the extension is in a pre-defined list of extensions. Which, as of now, only includes `txt`, `frag`, and `vert`
2. Another check for if it's an `svg`

In all these cases. Since there is a finite list of traditional extensions, we can safely change the function to return `null` if there isn't an extension and know we will just pass over these checks. As for dot-files (even though they don't really matter here), we'd still return the filename as the "extension" as before.